.github/workflows/release.yml

```
@@ -2,8 +2,8 @@ name: Release
 run-name: Release ${{ github.ref_name }}#${{ github.run_number }}
 on:
   workflow_run:
-    workflows: ['CI Pipeline']
-    branches: [main, alpha, beta, next]
+    workflows: [ 'CI Pipeline' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)